mowgli_etl/pipeline/wdc/wdc_offers_corpus.py

```diff
@@ -0,0 +1,26 @@
+from dataclasses import dataclass
+from typing import Optional, List
+from pathlib import Path
+from mowgli_etl.pipeline.wdc.wdc_offers_corpus_entry import WdcOffersCorpusEntry
+
+
+@dataclass(init=False)
+class WdcOffersCorpus:
+	entries: Optional[List[WdcOffersCorpusEntry]]
+
+	def __init__(self, *, wdc_json_file_path: Path):
+		self.entries = []
+		for row in open(wdc_json_file_path):
+			self.entries.append(WdcOffersCorpusEntry.from_json(row))
+
+	def sample(self, n: int):
+		if n < 1:
+			print(f"ERROR: Desired number of lines {n} is not a positive integer")
```

Comment:
raise ValueError instead of printing a message and returning None for exceptional conditions